src/Shouldly.Tests/ShouldBe/EnumerableType/NonRetraversableEnumerable.cs

```diff
@@ -10,22 +10,20 @@ public class NonRetraversableEnumerable
         private static IEnumerable<int> CreateTestEnumerable()
         {
             var foo = new TestEnumerable(new[] {
-                new[] {1, 2, 3 },
-                new[] {4, 5, 6 },
-                new[] {7, 8, 9 } });
+                new[] { 1, 2, 3 },
+                new[] { 4, 5, 6 },
+                new[] { 7, 8, 9 } });
 
             return foo.ReadLine();
         }
 
         [Fact]
         public void ActualEnumerableShouldOnlyBeTraversedOnce()
         {
-            var fooEnum = CreateTestEnumerable();
-
-            Verify.ShouldFail(() =>
-                fooEnum.ShouldBe(new [] {3, 2, 1 }),
+            Verify.ShouldFail(
+                () => CreateTestEnumerable().ShouldBe(new[] { 3, 2, 1 }),
```

Comment:
If we wanted, we could make it more likely for contributors to avoid the pitfall of keeping mutable state outside the `Verify.ShouldFail` call by making all the code they are extrapolating from use `static () =>`.